tracks/cpp/exercises/space-age/mentoring.md

```diff
@@ -0,0 +1,124 @@
+### Reasonable solutions
+```cpp
+#if !defined(SPACE_AGE_H)
+#define SPACE_AGE_H
+
+#include <cstdint>
+
+namespace space_age
+{
+    class space_age final
+    {
+    public:
+        explicit space_age(const std::int64_t seconds);
+        
+        std::int64_t seconds() const;
+        double on_earth() const;
+        double on_mercury() const;
+        double on_venus() const;
+        double on_mars() const;
+        double on_jupiter() const;
+        double on_saturn() const;
+        double on_uranus() const;
+        double on_neptune() const;
+
+    private:
+        enum class planet : std::int64_t;
+        double on_planet(const planet p) const;
+        std::int64_t _seconds;
+    };
+}
+
+#endif // SPACE_AGE_H
+```
+```cpp
+#include "space_age.h"
+
+namespace space_age
+{
+    enum class space_age::planet : std::int64_t
+    {
+        Earth = 31557600,
+        Mercury = 7600544,
+        Venus = 19414149,
+        Mars = 59355036,
+        Jupiter = 374355660,
+        Saturn = 929292360,
+        Uranus = 2651370000,
+        Neptune = 5200418600,
```

Comment:
Should we expect users to convert everything to seconds or stick with units of "Earth years"? Or does it even matter?

Reply:
I don't think it matters.
One think that can be improved here is adding a unit to this values - seconds in this case.
I do wander if there is a way to use std::chrono for this, as it makes the units nice and clear, and allows converting easily.